cvat-core/src/api.ts

```diff
@@ -374,6 +375,27 @@ function build(): CVATCore {
                 },
             },
         },
+        requests: {
+            async list(): Promise<{
+                requests: Request[];
+                count: number;
+            }> {
```

Comment:
For the most of resources we have returning lists like: PaginatedResource<Resource> where `count` is an additional property of returning array.

But for lambda manager and request manager we use another approach.

I believe it should be unified.